frontend/src/pages/Slot.tsx

```diff
@@ -108,6 +108,12 @@ const Slot = () => {
         <ResponsiveLayout>
             <Box sx={{ flexGrow: 1, marginTop: 4 }}>
                 <TitleSection title="Slot Overview" />
+                <Typography variant="body1" textAlign="left" paddingTop="20px" color="#051016" lineHeight="180%">
+                    The slot overview is designed to give you information about the activity around the current blockchain's state.
+                    It gives a look ahead to the next 16 slots and shows when a registered validator is scheduled.
```

Comment:
It actually gives a look ahead to the next 16-32 slots

Reply:
For now, it's only the next 16 slots for the current epoch. I thought we could add the 32 when the next epochs is included?

Reply:
Ah makes sense. Yes